pesterchum.py

```diff
@@ -3347,7 +3347,7 @@ def updateOptions(self):
             if opvmesssetting != curopvmess:
                 self.config.set("opvMessages", opvmesssetting)
             # animated smiles
-            animatesetting = self.optionmenu.animationscheck.isChecked()
+            animatesetting = self.optionmenu.animationscheck.isChecked() if not ostools.isOSXBundle() else False
```

Comment:
Might be better not to tie it to macOS explicitly, for if the behavior of animated smilies on mac is changed in the future.
```suggestion
            if hasattr(self.optionmenu, 'animationscheck'):
                animatesetting = self.optionmenu.animationscheck.isChecked()
            else:
                animatesetting = False
```
Think this ought to be equivalent?